syfertext/language.py

```diff
@@ -101,54 +101,31 @@ def make_doc(self, text: Union[str, String, StringPointer]):
         are stored in a Doc object which is then returned.
         """
 
-        # TODO: huge bugs in this function. deal with the case when the function a local string `text`
-        #       after it has been already called with a remote string
-        if self.tokenizer is None:
+        # Get the location ID of the machine that owns this Tokenizer
```

Comment:
Here we are getting the  ID of the 'worker' where the string is located not the machine that owns the Tokenizer.